projects/plugins/jetpack/_inc/client/at-a-glance/boost/index.jsx

```diff
@@ -13,35 +35,535 @@ const BOOST_PLUGIN_FILES = [
+							</tr>
+						</tbody>
+					</table>
+					<table>
+						<tbody>
+							<tr>
+								<th>D</th>
+								<td>35-50</td>
+							</tr>
+							<tr>
+								<th>E</th>
+								<td>25-35</td>
+							</tr>
+							<tr>
+								<th>F</th>
+								<td>0-25</td>
+							</tr>
+						</tbody>
+					</table>
+				</div>
+			</InfoPopover>
+		</div>
+	);
+};
+
+const ConversionLossPopover = () => {
+	const trackInfoClick = useCallback( () => {
+		analytics.tracks.recordJetpackClick( {
+			target: 'boost-conversion-loss-info-button',
+			feature: BOOST_PLUGIN_SLUG,
+		} );
+	}, [] );
+
+	const trackSourceClick = useCallback( () => {
+		analytics.tracks.recordJetpackClick( {
+			target: 'boost-conversion-loss-info-source',
+			feature: BOOST_PLUGIN_SLUG,
+		} );
+	}, [] );
+
+	return (
+		<div className="boost-conversion-loss-info">
+			<InfoPopover
+				onClick={ trackInfoClick }
+				screenReaderText={ __( 'Learn more about how slow sites lose visitors', 'jetpack' ) }
+			>
+				<p className="boost-conversion-loss-info__source">
+					{ __( 'Source: ', 'jetpack' ) }
+					<ExternalLink
+						href="https://web.dev/why-speed-matters/"
+						target="_blank"
+						rel="noopener noreferrer"
+						onClick={ trackSourceClick }
+					>
+						web.dev
+					</ExternalLink>
+				</p>
+			</InfoPopover>
+		</div>
+	);
+};
+
+const CriticalCssInfoPopover = () => {
+	const criticalCssUrl = getRedirectUrl( 'jetpack-boost-critical-css' );
+
+	const trackInfoClick = useCallback( () => {
+		analytics.tracks.recordJetpackClick( {
+			target: 'boost-critical-css-info-button',
+			feature: BOOST_PLUGIN_SLUG,
+		} );
+	}, [] );
+
+	const trackCriticalCSSLinkClick = useCallback( () => {
+		analytics.tracks.recordJetpackClick( {
+			target: 'boost-critical-css-info-link',
+			feature: BOOST_PLUGIN_SLUG,
+		} );
+	}, [] );
+
+	return (
+		<div className="boost-critical-css-info">
+			<InfoPopover
+				onClick={ trackInfoClick }
+				screenReaderText={ __( 'Learn more about how critical CSS works', 'jetpack' ) }
+			>
+				<h3 className="boost-critical-css-info__title">
+					{ __( 'Regenerate Critical CSS', 'jetpack' ) }
+				</h3>
+				<p>
+					{ createInterpolateElement(
+						__(
+							'You should regenerate <ExternalLink>Critical CSS</ExternalLink> to optimize speed whenever your site’s HTML or CSS structure changes after:',
+							'jetpack'
+						),
+						{
+							ExternalLink: (
+								<ExternalLink
+									onClick={ trackCriticalCSSLinkClick }
+									href={ criticalCssUrl }
+									target="_blank"
+									rel="noopener noreferrer"
+								/>
+							),
+						}
+					) }
+				</p>
+				<ul className="boost-critical-css-info__list">
+					<li>{ __( 'Making theme changes.', 'jetpack' ) }</li>
+					<li>{ __( 'Writing a new post/page', 'jetpack' ) }</li>
+					<li>{ __( 'Editing a post/page', 'jetpack' ) }</li>
+					<li>
+						{ __(
+							'Activating, Deactivating, or updating plugins that impact your site layout.',
```

Comment:
I don't think this D in Deactivating should be capitalized.

```suggestion
							'Activating, deactivating, or updating plugins that impact your site layout',
```